docs/en/04_Changelogs/4.6.0.md

```diff
@@ -75,6 +75,17 @@ module, you should upgrade it as well.
 
 Read [Allowed file types](Developer_Guides/Files/Allowed_file_types) in the Silverstripe CMS doc for all the details.
 
+## File status icons in the file manager
+
+![File status icons](https://raw.githubusercontent.com/silverstripe/silverstripe-framework/blob/4/docs/en/04_Changelogs/_images/file-status-icons.png "File status icons")
```

Comment:
Alt text should be descriptive, as it's what a screenreader will speak in place of the user seeing the image. There are some [useful articles around](https://moz.com/learn/seo/alt-text) if you're curious.

```suggestion
![File status icons](https://raw.githubusercontent.com/silverstripe/silverstripe-framework/blob/4/docs/en/04_Changelogs/_images/file-status-icons.png "A screenshot of file status icons being displayed in the file manager")
```